site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+
+{% Video src="video/rULxC7pPw3PFS4o9xr7v8isFmCv1/5IRBMSsCZwihnVIVK4gS.mp4",
+         controls=true,
+         poster="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/4ttTTum2aZuNZL04vYmI.jpg" %}
+
+Most people publishing signed exchanges today are using Cloudflare's amazingly easy-to-use [Automatic Signed Exchanges](https://support.cloudflare.com/hc/en-us/articles/4411075595661-Automatic-Signed-Exchanges-SXGs-Beta) (ASX) feature (though [open source options](https://web.dev/signed-exchanges/#tooling) exist, too):
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/Jok2O4SHNgW5d9y20YQP.png",
+       alt="Cloudflare settings panel with checkbox to enable Automatic Signed Exchanges",
+       width="800", height="373" %}
+
+In many cases, checking the box is enough to get the kind of drastic improvement shown above. However, sometimes there are a few more steps to ensure these signed exchanges are working as intended at each stage of the pipeline, and to optimize pages to take full advantage of prefetch.
+
+In the past couple of months since Cloudflare's launch, I've been reading and responding to questions on [various](https://support.google.com/webmasters/threads?hl=en&thread_filter=(%27%22signed%20exchange%22%27)) [forums](https://community.cloudflare.com/t/automatic-signed-exchanges-sxgs-beta-launch/312634) and advising sites on how to make sure they're getting the most out of their signed exchange deployments. In the process, I think I've learned enough to begin offering some common advice for frequently asked questions.
+
+Let's dig in.
+
+## Analyze
+
+One place to start is to use a lab tool like WebPageTest like shown above, so you can compare waterfalls—and LCP—with and without SXG prefetch.
+
+{% Aside 'caution' %}
+[Lab results may differ](https://web.dev/lab-and-field-data-differences/) from real user experiences, but they are a useful tool to diagnose what needs optimization and iterate quickly. We'll cover how to measure the resultant field data in the [Measure](#measure) section.
+{% endAside %}
+
+Generate a test without SXG in the standard way: go to [WebPageTest](https://www.webpagetest.org/) and enter a URL.
+
+  - Consider doing this while signed in, so you can more easily generate a comparison.
+  - Go to "Advanced Configuration" and choose a set of options there, rather than "Simple Configuration". (This makes it easy to assure that you have the same set of parameters as the "with SXG" test, which will use Advanced.)
+  - Under Browser, choose an Android phone under "Chrome Device Emulation". (This is also for parity with the "with SXG" test. As of writing, Google Search only supports SXG on Android.)
+  - Everything else is up to you. It may be helpful to increase "Number of Tests to Run" to 7, and pick a 4G connection.
```

Comment:
```suggestion
- Consider doing this while signed in, so you can more easily generate a comparison.
- Go to "Advanced Configuration" and choose a set of options there, rather than "Simple Configuration". (This makes it easy to assure that you have the same set of parameters as the "with SXG" test, which will use Advanced.)
- Under Browser, choose an Android phone under "Chrome Device Emulation". (This is also for parity with the "with SXG" test. As of writing, Google Search only supports SXG on Android.)
- Everything else is up to you. It may be helpful to increase "Number of Tests to Run" to 7, and pick a 4G connection.
```